src/pages/surveys/[id]/questions/[order].page.tsx

```diff
@@ -42,22 +53,34 @@ const Question: NextPage = () => {
 
   const totalQuestion = currentSurvey?.questions.length || 0;
 
+  const onSubmitResponses = () => {
```

Comment:
```suggestion
  const onSubmitResponse = () => {
```

Same suggestion for the `SurveyQuestion` component: 
- currentResponse **s**={response **s**}
- setResponse **s**={setResponse **s**}

Reply:
Updated in [b076636](https://github.com/hanam1ni/react-surveyor/pull/79/commits/b076636da5d3d95c56a7ea4528ad9ad708bceb65)